recipe/meta.yaml

```diff
@@ -1,35 +1,35 @@
 {% set name = "pluggy" %}
 {% set version = "1.0.0" %}
-{% set sha256 = "4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name [0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pluggy-{{ version }}.tar.gz
+  sha256: 4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159
 
 build:
-  number: 4
-  skip: true  # [py<35]
-  script: python -m pip install . --no-deps --ignore-installed
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 5
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.6
+    - setuptools-scm
     - pip
-    - setuptools_scm
   run:
-    - python
-    - importlib_metadata >=0.12  # [py<38]
+    - python >=3.6
```

Comment:
```suggestion
    - python >=3.8
```

Reply:
Actually according to [`setup.cfg`](https://github.com/pytest-dev/pluggy/blob/70fde45e41370327da008ff052e1da10aa714c0f/setup.cfg#L36), `python >=3.6` seems correct.

Reply:
Ahh I see, the suggestion is to make it no-arch (because then it can drop the dependency to `importlib-metadata`).

Not sure I agree with it though: there are still conda environments using Python 3.6 and 3.7, and implementing the changes here would render future releases of `pluggy` unusable on those Python versions, even if the upstream package still supports them (I understand conda-forge no longer builds py36 packages, but the point is still valid for py37).